### ISSUE TITLE
ZeroDivisionError for Running On-Demand All X instances  EC2 vCPU limit.

### ISSUE THREAD

**Issue by mahbub21463** [jantman/awslimitchecker#460]
## Bug Report

During version upgrade from `6.1.7` to `8.0.1`, we received `ZeroDivisionError` error for `Running On-Demand All X instances` limit of EC2 service . Current limit for `Running On-Demand All X instances` of EC2 service was 0 for our account.

### Stack trace
```
[ERROR] ZeroDivisionError: float division by zero
Traceback (most recent call last):
  File "/var/task/sentry_sdk/integrations/aws_lambda.py", line 46, in sentry_handler
    reraise(*exc_info)
  File "/var/task/sentry_sdk/_compat.py", line 54, in reraise
    raise value
  File "/var/task/sentry_sdk/integrations/aws_lambda.py", line 37, in sentry_handler
    return handler(event, context, *args, **kwargs)
  File "/var/task/bc_lambda/decorator.py", line 80, in wrapper
    _return = lambda_handler(event, context)
  File "/var/task/handler.py", line 18, in handle
    thresholds = aws_limit_checker.check_thresholds(use_ta=USE_TRUSTED_ADVISOR)
  File "/var/task/awslimitchecker/checker.py", line 633, in check_thresholds
    tmp = cls.check_thresholds()
  File "/var/task/awslimitchecker/services/base.py", line 260, in check_thresholds
    if limit.check_thresholds() is False:
  File "/var/task/awslimitchecker/limit.py", line 434, in check_thresholds
    pct = (usage / (limit * 1.0)) * 100
```

### Current limit for `Running On-Demand All X instances`

<img width="1419" alt="Screenshot 2020-03-03 at 2 38 30 AM" src="https://user-images.githubusercontent.com/29938552/75715745-1d36d900-5cf8-11ea-8727-aa16ecfdba25.png">

Entirely possible I'm missing something, but I thought it was worth raising.

### Version 8.0.1

**Comment by jantman** [comment]
@mahbub21463 I believe this issue gave the exact information that I was looking for in #458. I should have a fix released within the next day.

**Comment by jantman** [comment]
A fix for this, thanks to #458, has been released as [8.0.2](https://github.com/jantman/awslimitchecker/releases/tag/8.0.2) and is now [live on PyPI](https://pypi.org/project/awslimitchecker/).
